src/common/regions/regions_data.ts

```diff
@@ -94,7 +94,7 @@ function buildMetroAreas(countiesByFips: Dictionary<County>): MetroArea[] {
       const [name, statesText] = metro.cbsaTitle.split(', ');
       const stateCodes = statesText.split('-');
       return new MetroArea(
-        name,
+        `${name.split('-')[0]} metro area`,
```

Comment:
Maybe it would be better to store the name as is, and have an accessor method that returns the name of the main city with "metro area"?

```ts
const metro = new MetroArea('New York-Newark-Jersey City', ... );

// New York Metro Area
metro.metroName;
```

Some of the mocks had the names of all the cities under the location title, I think we might need them. Can you confirm capitalization of "New York Metro Area" with Karen? I'm not sure if we want `metro area` or `Metro Area` (that can be fixed in the rendering logic as well)